tests/fig41-full-adder/README.md

```diff
@@ -0,0 +1,17 @@
+## Adder tests
```

Comment:
This test is already at https://github.com/SymbiFlow/python-symbiflow-v2x/tree/master/tests/vtr/full-adder